pubspec.yaml

```diff
@@ -97,11 +97,11 @@ dev_dependencies:
   drift_dev: ^2.5.2
   fake_async: ^1.3.1
   flutter_checks: ^0.1.1
-  flutter_lints: ^4.0.0
+  flutter_lints: ^5.0.0
```

Comment:
By nature if this doesn't cause CI to fail, it can't have broken anything.

There is one interesting change:
> * Removes the following lints (see https://github.com/dart-lang/lints/issues 205):
>   * `prefer_const_constructors`
>   * `prefer_const_declarations`
>   * `prefer_const_literals_to_create_immutables`

So that ends the regular nagging from the analyzer to add `const` everywhere. As discussed in the thread, it's not clear that the performance benefit of it is/was material.